src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -2431,7 +2439,13 @@ encode %{
   enc_class riscv_enc_call_epilog() %{
     if (VerifyStackAtCalls) {
       // Check that stack depth is unchanged: find majik cookie on stack
-      __ call_Unimplemented();
+      int framesize = ra_->reg2offset_unchecked(OptoReg::add(ra_->_matcher._old_SP, -3 * VMRegImpl::slots_per_word));
+      Label CHECK_PASSED;
```

Comment:
Nit: We rarely use big names for labels. Maybe `Label stack_ok`?

Reply:
Yea, sure.

Personally I prefer labels to stand out as control transfer is a bit tricky to keep track of in assembly.